### ISSUE TITLE
Feature Request: Add the ability to specify a virtual network 

### ISSUE THREAD

**Issue by rusher81572** [scylladb/scylla-grafana-monitoring#312]
Hello, I think it would be good to add an option to start-all.sh that enables the user to specify a Docker virtual network for the Prometheus and Grafana containers. This would greatly help the Mutant Monitoring Blog Series that uses a virtual network for the Scylla containers.

**Comment by rusher81572** [comment]
The docker command is like this for a container:
```
docker run -d  --net=mms_web --name container-name container
```
I think start-all.sh can have a -n or --network option

**Comment by rusher81572** [comment]
That will not work because we are running 3 Scylla nodes. --net=host would put all the Scylla nodes on the host interface with the same IP.

**Comment by tzach** [comment]
@amnonh can you help? the best will be to find a way to propagate any list of parameters to each of the 3 docker run command

**Comment by tzach** [comment]
@amnonh ping

**Comment by amnonh** [comment]
Just sent a pull request for a general capability to add container
parameter option

**Comment by tzach** [comment]
fixed by #314
